go.mod

```diff
@@ -1,18 +1,59 @@
 module github.com/fyne-io/fyne-cross
 
-go 1.14
+go 1.17
 
 require (
 	github.com/BurntSushi/toml v1.1.0
 	github.com/Kodeworks/golang-image-ico v0.0.0-20141118225523-73f0f4cfade9
-	github.com/andybalholm/brotli v1.0.4 // indirect
-	github.com/fyne-io/fyne-cross/internal/cloud v0.0.0
-	github.com/golang/snappy v0.0.4 // indirect
-	github.com/pierrec/lz4/v4 v4.1.14 // indirect
+	github.com/aws/aws-sdk-go v1.41.19
+	github.com/klauspost/compress v1.13.4
+	github.com/mholt/archiver/v3 v3.5.1
```

Comment:
The bump to Go 1.17 should let you bump both of these versions to their latest versions, I think.

Reply:
Bumping those require code change I think. I can see if there is some minor bump possible that won't require code change.

Reply:
The bump of compress should not need any code changes but it will not let us compile with older versions of Go though.